modules/vba_documents/app/sidekiq/vba_documents/upload_processor.rb

```diff
@@ -60,14 +66,11 @@ def download_and_process
         validate_metadata(parts[META_PART_NAME], @upload.consumer_id, @upload.guid,
                           submission_version: @upload.metadata['version'].to_i)
         metadata = perfect_metadata(@upload, parts, timestamp)
+        metadata['ICN'] = icn if icn.present?
 
         pdf_validator_options = VBADocuments::DocumentRequestValidator.pdf_validator_options
         validate_documents(parts, pdf_validator_options)
 
-        # attempt to use consumer's file number field to look up the claiments ICN
-        icn = find_icn(metadata['fileNumber'])
-        metadata['ICN'] = icn if icn.present?
-
```

Comment:
Here is my mistake in the original code, i guess i thought that the metadata hash above was the same metadata we store in our tracking record. It's not, the metadata below is dynamic, we build it in memory and send it to EMMS as part of our req to them.